### PR TITLE
SEO

### PR DESCRIPTION
**Pull request by rsmalarz**

### CODE REVIEW

**Review by fkops — APPROVED** [review]

File: src/components/CookiesAlert/CookiesAlert.js [hunk 1]
```diff
@@ -20,9 +18,9 @@ export const CookiesAlert = ({
   content, confirmCookies, isVisible,
 }) => (
   <Wrapper isVisible={isVisible}>
-    <SectionHeading>
+    <CookieHeading>
```
Comment (kjhank) [inline src/components/CookiesAlert/CookiesAlert.js:21]: Why not just use the `as="p"` prop instead of duplicating the entire component under another name?

**Review by kjhank — CHANGES_REQUESTED** [review]

File: src/components/CookiesAlert/CookiesAlert.js (hunk 1 above)
Comment (rsmalarz) [reply src/components/CookiesAlert/CookiesAlert.js:21]: Yes, you're right we could use the as="p" prop, but we have talked to the client, and he is fine with it.

Comment (kjhank) [reply src/components/CookiesAlert/CookiesAlert.js:21]: Sure, when the code is on the client's repo, they'll have a say in regards to code quality. Until then - [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

**Comment by fkops** [comment]
Good morning,

can we just agree to Krzysztof’s idea regarding the issue with the pull request and implement SEO changes on to the site?

Kind regards
Filip Kopijer
Digital Marketing Manager

***@***.***

Przedsiębiorstwo Farmaceutyczne LEK-AM Sp. z o.o.
Biuro Handlowe
Al. Jana Pawła II 80, 00-175 Warszawa

***@***.******@***.***>
Kom: +48728432939
_____________________________________________

File: src/components/CookiesAlert/CookiesAlert.js (hunk 1 above)
Comment (fkops) [reply src/components/CookiesAlert/CookiesAlert.js:21]: @rsmalarz please adhere to kjhank comment and correct pull request in order to commit. Thank you.

Comment (fkops) [reply src/components/CookiesAlert/CookiesAlert.js:21]: @kjhank can you change the pull request as required to comply with DRY code quality?

**Review by kjhank — APPROVED** [review]

**Pull request merged**